environment.doc.yml

```diff
@@ -3,12 +3,13 @@ channels:
     - defaults
     - conda-forge
 dependencies:
-    - lightgbm=3.2.1
     - numpydoc=1.1.0
-    - pandas=1.3.5
-    - python=3.10
-    - scikit-learn
     - sphinx=4.3.2
     - sphinx-gallery=0.10.1
     - sphinx_rtd_theme=1.0.0
     - typing_extensions=4.0.1
+    - pandas
```

Comment:
why did you remove the versions ?

Reply:
because there's no incompatibility with the latest version of pandas so far. so there's no reason to constrain the version

Reply:
I still feel like we should force a version. We don't want the doc not building for a version problem. (we can perform version updates every now and then?)